Hyper/PlayerData/InventorySystem/Items/Item.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Hyper.PlayerData.InventorySystem.Items;
 
-internal abstract class Item
+internal abstract class Item : IEquatable<Item>
```

Comment:
where is this used?

Reply:
`Scene.TryPickLamp` adds a new lamp item to the inventory, and up until now items were compared based on reference equality which resulted in lamp items not aggregating correctly, now they are compared on id